sources/packages/web/src/components/students/applicationTracker/Completed.vue

```diff
@@ -21,7 +21,15 @@
     content="Your institution is proposing changes to your application. Please review the details and allow (or decline) the proposed change from your institution. Allowing the change will enable StudentAid BC to make a final decision on the proposed changes, which will result in a new assessment if the changes are approved. Please review the changes carefully and contact your institution if you require more information."
   >
     <template #actions>
-      <v-btn color="primary">Review changes</v-btn>
+      <v-btn
+        color="primary"
+        @click="
+          viewApplicationOfferingChangeRequest(
+            assessmentDetails.applicationOfferingChangeRequestId,
```

Comment:
i have the below error in my local, 
![image](https://github.com/bcgov/SIMS/assets/77353155/1f7e3f17-80c1-4287-a39b-3c34bd022a6e)